util/topgen/templates/meson.build.tpl

```diff
@@ -0,0 +1,39 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+${gencmd.replace("//", "#")}
+<%text>
```

Comment:
Why do you need a %text escaping directive here?

Reply:
But not for the license? I think there's something wrong here. Is it the multiple ##s?

Reply:
Fixed.